tools/getfile/Readme.md

```diff
@@ -0,0 +1,56 @@
+# How to Use Getfile tool
+Here we explain how to enable build support for GETFILE,  
+and how to use this function.
+
+## Contents
+> [Build Steps](#how-to-enable-getfile)  
+> [Use Steps](#how-to-use-getfile)  
+
+## How to enable RAMDUMP
```

Comment:
How to enable GETFILE

Please use same characters for name. Because SE team checks and takes it as fault.
GETFILE or getfile

Reply:
Has been modified